tutorials/navigation/navigation_using_navigationmeshes.rst

```diff
@@ -17,254 +19,320 @@ Navigation meshes are the only things considered when doing pathfinding, e.g. vi
-    Do not create multiple disconnected islands in the same NavigationRegion2D or NavigationPolygon resource if they should be later connected.
+        .. image:: img/nav_region_baking_01.webp
 
-For 2D, no similar navigation mesh baking with geometry parsing exists like in 3D.
-The Geometry2D class functions for offset, merge, intersect, and clip can be used to shrink or enlarge existing NavigationPolygons for different actor sizes.
+        In order for the region to work a :ref:`NavigationPolygon<class_NavigationPolygon>` resource needs to be added.
 
-Creating 3D navigation meshes
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+        The properties to parse and bake a navigation mesh are then part of the used resource and can be found in the resource Inspector.
 
-.. image:: img/baked_navmesh.png
+        .. image:: img/nav_region_baking_02.webp
 
-Navigation meshes in the 3D editor are created with the help of the
-:ref:`NavigationMeshGenerator<class_NavigationMeshGenerator>` singleton
-and the NavigationMesh bake settings that appear in the editor inspector.
+        The result of the source geometry parsing can be influenced with the following properties.
 
-Navigation mesh baking is the process of creating a simplified mesh used for pathfinding out of (complex) 3D level geometry.
-For this process Godot parses scene geometry and hands the raw mesh or collision data to the
-third-party ReCast library for processing and creation of the final navigation mesh.
+        - The ``parsed_geometry_type`` that filters if visual objects or physics objects or both should be parsed from the :ref:`SceneTree<class_SceneTree>`. For more detail what objects are parsed and how see the section about parsing source geometry further below.
+        - The ``collision_mask`` filters which physics collision objects are included when the 'parsed_geometry_type' includes static colliders.
+        - The ``source_geometry_mode`` that defines on which node(s) to start the parsing and how to traverse the :ref:`SceneTree<class_SceneTree>`.
+        - The ``source_geometry_group_name`` is used when only a certain node group should be parsed. Depends on the selected 'source_geometry_mode'.
 
-The resulting navigation mesh is an approximation of the source geometry surfaces
-for both performance and technical reasons. Do not expect the navigation mesh
-to perfectly follow the original surfaces. Especially navigation polygons placed
-over ramps will not keep an equal distance to the ground surface. To align an
-actor perfectly with the ground use other means like physics.
+        With the source geometry added the result of the baking can be influenced with the following properties.
 
-.. warning::
+        - The ``cell_size`` sets the rasterization grid cells and should match the navigation map size.
+        - The ``agent_radius`` that shrinks the baked navigation mesh to have enough margin for the agent (collision) size.
```

Comment:
```suggestion
        - The ``agent_radius`` shrinks the baked navigation mesh to have enough margin for the agent (collision) size.
```
Redundant IMO, otherwise use it above for consistency